### PR TITLE
Cve 2019 14905

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14905.json [hunk 1]
```diff
@@ -4,21 +4,21 @@
...
         "ID": "CVE-2019-14905",
-        "ASSIGNER": "psampaio@redhat.com"
+        "ASSIGNER": "darunesh@redhat.com"
     },
     "affects": {
         "vendor": {
             "vendor_data": [
                 {
-                    "vendor_name": "Red Hat",
+                    "vendor_name": "[UNKNOWN]",
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14905.json]: Why?

File: 2019/14xxx/CVE-2019-14905.json [hunk 2]
```diff
@@ -4,21 +4,21 @@
...
                     "product": {
                         "product_data": [
                             {
                                 "product_name": "Ansible",
                                 "version": {
                                     "version_data": [
```
Comment (cve-team) [inline 2019/14xxx/CVE-2019-14905.json]: The reference indicates that versions before Ansible 2.9.4, 2.8.8, and 2.7.16 are affected. If this is the case, please make the corrections in the version_data field to reflect the correct affected versions.

**Comment by ccoffin** [comment]
Red Hat CNA,

We are currently waiting for the iamleot and cve-team comments to be addressed before populating this CVE entry. Let us know if you need anything or have any questions.

Regards,

Chris

File: 2019/14xxx/CVE-2019-14905.json (hunk 2 above)
Comment (iamleot) [reply 2019/14xxx/CVE-2019-14905.json]: AFAICS I think this is fixed respectively in 2.9.3, 2.8.8 and 2.7.16:

 https://github.com/ansible/ansible/blob/stable-2.9/changelogs/CHANGELOG-v2.9.rst
 https://github.com/ansible/ansible/blob/stable-2.8/changelogs/CHANGELOG-v2.8.rst
 https://github.com/ansible/ansible/blob/stable-2.7/changelogs/CHANGELOG-v2.7.rst

File: 2019/14xxx/CVE-2019-14905.json [hunk 3]
```diff
@@ -4,21 +4,21 @@
...
                                         {
-                                            "version_value": "2.9.2"
+                                            "version_value": "2.9.4"
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14905.json]: Shouldn't this be 2.9.3?:

 https://github.com/ansible/ansible/blob/stable-2.9/changelogs/CHANGELOG-v2.9.rst

File: 2019/14xxx/CVE-2019-14905.json [hunk 4]
```diff
@@ -68,7 +68,7 @@
         "description_data": [
             {
                 "lang": "eng",
-                "value": "A vulnerability in Ansible's nxos_file_copy module can be used to copy files to a flash or bootflash on NXOS devices. Malicious code could craft the filename parameter to perform OS command injections. This could result in a loss of confidentiality of the system among other issues."
+                "value": "A vulnerability was found in Ansible engine before 2.9.2, 2.8.8 and 2.7.16, where Ansible's nxos_file_copy module can be used to copy files to a flash or bootflash on NXOS devices. Malicious code could craft the filename parameter to perform OS command injections. This could result in a loss of confidentiality of the system among other issues."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14905.json]: This doesn't seem correct.

AFAICS it was fixed in 2.9.3, not 2.9.2:

 https://github.com/ansible/ansible/blob/stable-2.9/changelogs/CHANGELOG-v2.9.rst

**Comment by cristina479** [comment]
Red Hat CNA,

Can you address the comments in this PR  in order to publish this CVE ID to the CVE List?

Thank you,
Cristina

File: 2019/14xxx/CVE-2019-14905.json [hunk 5]
```diff
@@ -68,7 +68,7 @@
         "description_data": [
             {
                 "lang": "eng",
-                "value": "A vulnerability in Ansible's nxos_file_copy module can be used to copy files to a flash or bootflash on NXOS devices. Malicious code could craft the filename parameter to perform OS command injections. This could result in a loss of confidentiality of the system among other issues. Ansible Engine before 2.9.2, 2.8.8 and 2.7.16 are affected."
+                "value": "A vulnerability was found in Ansible Engine 2.7.15, 2.8.7, 2.9.2 and before, where in Ansible's nxos_file_copy module can be used to copy files to a flash or bootflash on NXOS devices. Malicious code could craft the filename parameter to perform OS command injections. This could result in a loss of confidentiality of the system among other issues."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14905.json]: Please change the description from:

 `[...] Ansible Engine 2.7.15, 2.8.7, 2.9.2 and before [...]`

to:

 `[...] Ansible Engine versions 2.9.x before 2.9.3, 2.8.x before 2.8.8, 2.7.x before 2.7.16 and earlier [...]`

**Pull request merged**